yjit/src/asm/mod.rs

```diff
@@ -370,4 +403,12 @@ impl OutlinedCb
     pub fn unwrap<'a> (&'a mut self) -> &'a mut CodeBlock {
         &mut self.cb
     }
+
+    /// Comments on an outlined code block are no-ops
+    pub fn add_comment(&mut self, comment: String) {
+    }
+
+    /// Comments on an outlined code block are no-ops
+    pub fn add_comment_at(&mut self, pos: usize, comment: String) {
+    }
```

Comment:
Outlined code blocks need to be unwrapped to be used, so these methods will never get called.

If we're storing the asm comments on the code blocks themselves, this makes it possible to store comments on outlined code blocks as well.